tests_integration/containers/docker/bootstrap_containers_setup.sh

```diff
@@ -0,0 +1,4 @@
+cd ..
+docker build -t snowpark_test:1 containers/docker
```

Comment:
does the snowpark_test:1 needs the repository url before it?
(e.g. myorg-myacct.registry.snowflake.com/my_db/my_schema/my_repository/snowpark_test:1)

Reply:
Fixed